### ISSUE TITLE
Still maintained? 

### ISSUE THREAD

**Issue by sabbelasichon** [jainishsenjaliya/js_faq#19]
Would you be interested if i would add support for TYPO3 9 and 10?

**Comment by jainishsenjaliya** [comment]
Kindly contact me on skype : jainishsenjaliya

**Comment by sabbelasichon** [comment]
@jainishsenjaliya Do you have a slack account under the TYPO3 organization? I am not using Skype anymore. Sorry.

**Comment by flightsupport** [comment]
@sabbelasichon I'd be interested in Typo3 v9+ support. Did you start working on it?

**Comment by sabbelasichon** [comment]
@flightsupport Yes. It is v10 compatible but with https://github.com/FriendsOfTYPO3/typo3db_legacy as requirement. But it is not published. Maybe you should ask / sponsoring @jainishsenjaliya

**Comment by flightsupport** [comment]
> @flightsupport Yes. It is v10 compatible but with https://github.com/FriendsOfTYPO3/typo3db_legacy as requirement. But it is not published. Maybe you should ask / sponsoring @jainishsenjaliya

Nice. Thanks for the quick tip 👍
